resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -268,27 +268,19 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
         .getOrElse {
           throw new SparkException(s"Executor Spark home `$EXECUTOR_HOME` is not set!")
         }
-      val runScript = new File(executorSparkHome, "./bin/spark-class").getPath
-      command.setValue(
-        "%s \"%s\" org.apache.spark.executor.CoarseGrainedExecutorBackend"
-          .format(prefixEnv, runScript) +
-        s" --driver-url $driverURL" +
-        s" --executor-id $taskId" +
-        s" --hostname ${executorHostname(offer)}" +
-        s" --cores $numCores" +
-        s" --app-id $appId")
+      val executable = new File(executorSparkHome, "./bin/spark-class").getPath
+      val runScript = "%s \"%s\" org.apache.spark.executor.CoarseGrainedExecutorBackend"
```

Comment:
(The string interpolation was fine, but just leave it, no point changing further)

Reply:
There was a problem with interpolation - checkstyle was failing on the following string:
```
val runScript = s"$prefixEnv \"$executable\" org.apache.spark.executor.CoarseGrainedExecutorBackend"
```
Error:
```
error file=/Users/akirillov/projects/mesosphere/spark/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala message=Expected token SEMI but got Token(STRING_LITERAL," org.apache.spark.executor.CoarseGrainedExecutorBackend",10094," org.apache.spark.executor.CoarseGrainedExecutorBackend")
```

The working variant with interpolation looks as following:
```
val runScript =
        s"""$prefixEnv "$executable"
           | org.apache.spark.executor.CoarseGrainedExecutorBackend
           |""".stripMargin.replaceAll("\n", " ")
```
not sure if it's better than `String.format`

Reply:
Oh, you might need `${executable}` in this case, but it doesn't matter enough to bother with.

Reply:
Ah, thanks for the pointer. Updated it alongside with too long test description which failed checkstyle.

Reply:
And `\"${executable}\$` fails to compile. Reverted back to `String.format`, squashed commits, and rebased on master.

Reply:
Weird, OK. Maybe you need the `s"""..."""` syntax in this case but I wouldn't bother.